app/Website/src/Http/Requests/SubscribeRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Website\Http\Requests;
+
+use Illuminate\Validation\Rule;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rules\RequiredIf;
+
+
+class SubscribeRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'first_name' => ['required', 'string', 'max:20'],
```

Comment:
Do we only ask for first name, or full name?

Reply:
After further consultations, only the first name is required.